### PR TITLE
💄 Functions Rework Composition Section

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Rework the Composition of functions section

### Why
It's better this way.

### CODE REVIEW

File: site/topics/functions/functions.rst [hunk 1]
```diff
@@ -374,26 +374,37 @@ Format of a Function
...
-* We can nest functions, too:
-    >>> do_stuff(do_stuff(2, 2), do_stuff(2, 2))
-    72
-* If you get confused tracing nested functions, just remember:
-    * Functions get *evaluted* and turned into values
-    * Find a function you can evaluate
-    * Evaluate it
-    * Cross out the function and replace it with the *value* it returns
-    * Keep doing this until you're down to one value.
+* We can compose functions of other functions
```
Comment (twentylemon) [inline site/topics/functions/functions.rst:377]: of -> from ? from seems to fit better

**Review by twentylemon — APPROVED** [review]

File: site/topics/functions/functions.rst (hunk 1 above)
Comment (jameshughes89) [reply site/topics/functions/functions.rst:377]: hmmmmmmmmmm. I don't disagree, but I also don't disagree with the other

Comment (jameshughes89) [reply site/topics/functions/functions.rst:377]: how about

`* Functions can be composed of other functions`

Comment (twentylemon) [reply site/topics/functions/functions.rst:377]: Still _from_ fits a little better, imo.

Comment (jameshughes89) [reply site/topics/functions/functions.rst:377]: Is the following a reasonable comparison: _This cake is composed of flower, water, sugar, salt..._

Comment (jameshughes89) [reply site/topics/functions/functions.rst:377]: or is that _from_ too? I really don't know.

Comment (twentylemon) [reply site/topics/functions/functions.rst:377]: /shrug

maybe students will open a pr

Comment (jameshughes89) [reply site/topics/functions/functions.rst:377]: According to people on the internet, (it's of)[https://english.stackexchange.com/questions/56985/what-should-i-use-composed-of-or-composed-from#:~:text=Composed%20of%20is%20standard%20from,some%20other%20such%20contextual%20situation.&text=Neither%20composed%20of%20nor%20composed%20from%20is%20a%20phrasal%20verb.].

I guess it's what they call a _collocation _ --- we just kinda' like using them together even though from is totally fine really.

Comment (twentylemon) [reply site/topics/functions/functions.rst:377]: I blame society.

**Pull request merged**